CRM/Contribute/WorkflowMessage/Contribution/BasicContribution.php

```diff
@@ -68,9 +112,68 @@ private function addExampleData(GenericWorkflowMessage $messageTemplate, $exampl
     }
     $mockOrder = new CRM_Financial_BAO_Order();
     $mockOrder->setTemplateContributionID(50);
-    $mockOrder->setPriceSetToDefault('contribution');
+    if (empty($example['is_show_line_items'])) {
+      $mockOrder->setPriceSetToDefault('contribution');
+      $mockOrder->setOverrideTotalAmount($contribution['total_amount']);
+      $mockOrder->setDefaultFinancialTypeID($contribution['financial_type_id']);
+    }
+    else {
+      $priceSet = $this->getNonQuickConfigPriceSet();
```

Comment:
`getSlowConfigPriceSet` doesn't sound right either. Ok. 🤷

Reply:
could be more accurate though?